src/PropertyManager/Travely.PropertyManager.API/Helpers/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Travely.PropertyManager.Data.EntityFramework;
+using Travely.PropertyManager.Service.Contracts;
+using Travely.PropertyManager.Service.Services;
+
+namespace Travely.PropertyManager.Bootstrapper.Helpers
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection ConfigureAutoMapper(this IServiceCollection services)
+        {
+            var mappingProfiles = new[]
+            {
+                typeof(Travely.PropertyManager.API.MappingProfiles.PropertyMappingProfile),
+                typeof(Travely.PropertyManager.Service.MappingProfiles.PropertyMappingProfile)
+            };
+
+            services.AddAutoMapper(mappingProfiles); 
+
+            return services;
+        }
+
+        public static IServiceCollection ConfigureDbContext(this IServiceCollection services, string connectionString)
+        {
+            services.AddDbContext<PropertyDbContext>(options =>
+                 options.UseSqlServer(connectionString, x => x.MigrationsAssembly("Travely.PropertyManager.Data")));
+
+            return services;
+        }
+
+        public static IServiceCollection RegisterServices(this IServiceCollection services)
```

Comment:
What is the use case of this method ?

Reply:
registration of all services to IOC container will happen here